python-sdk/nuscenes/eval/detection/data_classes.py

```diff
@@ -369,10 +369,11 @@ def tp_errors(self) -> Dict[str, float]:
     @property
     def tp_scores(self) -> Dict[str, float]:
         scores = {}
+        tp_errors = self.tp_errors
         for metric_name in TP_METRICS:
 
             # We convert the true positive errors to "scores" by 1-error
-            score = 1.0 - self.tp_errors[metric_name]
+            score = 1.0 - tp_errors[metric_name]
```

Comment:
why did you make this change?

Reply:
In the old way, we will calculate all the `tp_errors` for each `metric_name`.